DEVELOPERS.md

```diff
@@ -1,18 +1,53 @@
 # Docs for Developers
 
-The entire monorepo is managed by [moon](https://moonrepo.dev/).
+The entire monorepo is managed by [Earthly](https://earthly.dev).
 And you need [direnv](https://direnv.net/).
 
-## How to install deps
-```shell
-go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.2
-go install github.com/onsi/ginkgo/v2/ginkgo@v2.8.3
-```
-
-
-## Generate SDK with Moon and Speakeasy
+## How to start developing
+To use Earthly and dev environments in Kubernetes, you need the following environment variables (in .direnv/.env):
+Environment variables are split into several pieces:
+- Project config: License of the various tools used in the repository
+    - SPEAKEASY_API_KEY: License for [Speakeasy](https://www.speakeasyapi.dev/)
+    - GITHUB_TOKEN: Github token for actions
+    - GORELEASER_KEY: Github token for releases
+- vCluster config:
```

Comment:
If vCluster is an integral part of the deployment, I feel like this should be expressed explicitly with some link to a documentation for setting this up. As far as I can tell from the vCluster docs, one could just as well just deploy to a barebones kubernetes cluster.